Subscriber/JavascriptSubscriber.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace MakairaConnect\Subscriber;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Enlight\Event\SubscriberInterface;
+
+class JavascriptSubscriber implements SubscriberInterface
+{
+
+    /**
+     * only add Events::<classes>
+     *
+     * @return array
+     */
+    public static function getSubscribedEvents(): array
+    {
+        return [
+            'Theme_Compiler_Collect_Plugin_Javascript', 'collectJs'
+        ];
+    }
+
+    public function collectJs(\Enlight_Event_EventArgs $args)
+    {
+        $jsFiles = glob(__DIR__ . '../Resources/frontend/js/*.js');
```

Comment:
`__DIR__` expands to the directory of the script without a tailing slash. It's possible that you forgot a slash `/`.

```suggestion
        $jsFiles = glob(__DIR__ . '/../Resources/frontend/js/*.js');
```

Reply:
You are right, the `__DIR__` constant doesn't have a trailing slash but my code is still working correctly. That's weird though.
Anyway, I fixed it.